src/site/stages/build/drupal/graphql/faqMultipleQa.graphql.js

```diff
@@ -97,7 +97,6 @@ fragment faqMultipleQA on NodeFaqMultipleQA {
 const GetNodeMultipleQaPages = `
   ${fragments.richTextCharLimit1000}
   ${fragments.reactWidget}
-  ${fragments.alertParagraph}
```

Comment:
I didn't magically know to remove this. The GraphQL query was returning with errors due to this fragment no longer being necessary (it was included in the `alertParagraphSingle` fragment but removed in this PR)

![image](https://user-images.githubusercontent.com/1915775/108402895-1a284780-71ec-11eb-8811-6fbfb6d0f3e8.png)